pkgs/tools/misc/vdirsyncer/default.nix

```diff
@@ -1,17 +1,17 @@
-{ stdenv, fetchurl, python3Packages, glibcLocales }:
+{ stdenv, fetchurl, python3Packages, glibcLocales, pkgs, rust, cargo }:
 
 # Packaging documentation at:
 # https://github.com/untitaker/vdirsyncer/blob/master/docs/packaging.rst
 let
   pythonPackages = python3Packages;
 in
 pythonPackages.buildPythonApplication rec {
-  version = "0.16.3";
+  version = "0.17.0a1";
```

Comment:
we typically do not have alpha's in Nixpkgs.

Reply:
Feel free to leave this unmerged (even after the remaining issues with this PR are solved) until I can bump to a new stable release (if there is any).